mcc/odin/odin/snapshot_list.go

```diff
@@ -1,12 +1,37 @@
 package odin
 
 import (
+	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/aws/aws-sdk-go/service/rds"
 	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
 )
 
+const (
+	// RFC8601 is the date/time format used by AWS.
+	RFC8601 = "2006-01-02T15:04:05-07:00"
```

Comment:
Seriously? I definitely prefer the strftime format, common things are obvious and every programmer or unix admin learned them anyway at some point :)

so, '01/15/02' (which looks like a totally fine date, january 15th 2002) would equal the format `%m-H-d`, right? :stuck_out_tongue_winking_eye: